manim/animation/movement.py

```diff
@@ -110,6 +110,21 @@ def interpolate_mobject(self, alpha: float) -> None:
 
 
 class MoveAlongPath(Animation):
+    """Make Mobject m1 move along path of Mobject m2.
+    Example
+    --------
+    .. manim:: MoveAlongPathExample
+
+        class MoveAlongPathExample(Scene):
+            def construct(self):
+                d1= Dot()
+                l1 = Line(LEFT,RIGHT)
+                l2= VMobject()
+                self.add(d1,l1,l2)
+                l2.add_updater(lambda x: x.become(Line(LEFT, d1.get_center()).set_color(ORANGE)))
+                self.play(MoveAlongPath(d1,l1), rate_func= linear)
```

Comment:
Just some inconsistencies in the spacing.

```suggestion
                d1 = Dot()
                l1 = Line(LEFT, RIGHT)
                l2 = VMobject()
                self.add(d1, l1, l2)
                l2.add_updater(lambda x: x.become(Line(LEFT, d1.get_center()).set_color(ORANGE)))
                self.play(MoveAlongPath(d1, l1), rate_func=linear)
```